default/scripting/techs/defense/mines.macros

```diff
@@ -9,8 +10,12 @@ EG_SYSTEM_MINES
                 ContainedBy And [
                     System 
                     Contains Or [
-                        Source
                         And [
```

Comment:
Given that most of this scope condition is reused numerous times in this macro, a sub-macro for it seems reasonable...